hw/lint/data/veriblelint.hjson

```diff
@@ -0,0 +1,24 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+    tool_srcs: []
+
+    // TODO(#1342): pass Verible config file to FuseSoC, once supported by Verible
+    // We rely on fusesoc to run lint for us
+    build_cmd:  "fusesoc"
+    build_opts: ["--cores-root {proj_root}",
+                 "run",
+                 "--target={flow}",
+                 "--tool={tool}",
+                 "--build-root={build_dir}",
+                 "{fusesoc_core}"]
+
+    // TODO(#1342): switch over to native structured tool output, once supported by Verible
+    // Verible lint-specific results parsing script that is called after running lint
+    report_cmd: "{proj_root}/hw/lint/tools/{tool}/parse-lint-report.py "
+    report_opts: ["--repdir={build_dir}",
+                  "--outdir={build_dir}"]
+
+    results_title_postfix: " Style Lint Results"
```

Comment:
Is it better to convert this to a flag-like structure:
```
  is_style_lint: True
```
Based on this value, you can update the behavior in the `LintCfg` class (at more than one place even, if needed).

Reply:
I was thinking that a postfix field would give us some more flexibility for future tools (we may want to add CDC or RDC lint passes in the future). But if you think we should just enumerate all these lint types I'll be happy to change that.

Reply:
I think you are right, especially if we want to customize the reports beyond just changing the title in the future. I've converted this to a flag.

Reply:
In any case, I am fine with either solution.